test/clients/azure/client.go

```diff
@@ -50,12 +59,12 @@ func NewClientFromEnvironment() (*Client, error) {
 	}
 
 	var rpURL string
-	focus := []byte(config.GinkgoConfig.FocusString)
+	focus := config.GinkgoConfig.FocusString
 	switch {
-	case adminRpFocus.Match(focus), fakeRpFocus.Match(focus):
+	case adminRpFocus.match(focus), fakeRpFocus.match(focus):
 		fmt.Println("configuring the fake resource provider")
 		rpURL = realfakerp.StartServer(log, conf, realfakerp.LocalHttpAddr)
```

Comment:
Btw, this is wrong, StartServer should ensure there is no fakerp running before trying to start a new one.

Reply:
Probably doesn't matter since we don't check the error returned from ListenAndServe.